### PR TITLE
🌐 Add Korean translation for `docs/ko/docs/tutorial/background-tasks.md`

### PR DESCRIPTION
**Pull request by junah201**
- Add Korean translation for `docs/ko/docs/tutorial/background-tasks.md`
- Related https://github.com/tiangolo/fastapi/issues/2017

### CODE REVIEW

File: docs/ko/docs/tutorial/background-tasks.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# 백그라운드 작업
+
+응답을 반환한 후 실행할 백그라운드 작업을 정의할 수 있습니다.
+
+백그라운드 작업은 클라이언트가 응답을 받기 위해 작업이 완료될 때까지 기다릴 필요가 없기 때문에 요청 후에 발생해야하는 작업에 매우 유용합니다.
+
+이러한 작업에는 다음이 포함됩니다.
+
+* 작업을 수행한 후 전송되는 이메일 알림
+    * 이메일 서버에 연결하고 이메일을 전송하는 것은 "느린" (몇 초) 경향이 있으므로 백그라운드에서 이메일을 보낼 수 있습니다.
```
Comment (joonas-yoon) [inline docs/ko/docs/tutorial/background-tasks.md:10]: Original text is:

> you can return the response right away and send the email notification in the background.

so I recommend to translate them into such as:

```suggestion
    * 이메일 서버에 연결하고 이메일을 전송하는 것은 (몇 초 정도) "느린" 경향이 있으므로, 응답은 즉시 반환하고 이메일 알림은 백그라운드에서 전송하는 게 가능합니다.
```

File: docs/ko/docs/tutorial/background-tasks.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+* 처리 데이터
```
Comment (joonas-yoon) [inline docs/ko/docs/tutorial/background-tasks.md:11]: I think this is written to as verb, so it is:

```suggestion
* 데이터 처리:
```

File: docs/ko/docs/tutorial/background-tasks.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+- 백그라운드에서 실행되는 작업 함수 (`write_notification`).
+- 작업 함수에 순서대로 전달되어야 하는 일련의 인자 (`email`).
+- 작업 함수에 전달되어야하는 모든 키워드 인자 (`message="some notification"`).
+
+## 의존성 주입
+
+`BackgroundTasks`를 의존성 주입 시스템과 함께 사용하면 _경로 동작 함수_, 종속성, 하위 종속성 등 여러 수준에서 BackgroundTasks 유형의 매개변수를 선언할 수 있습니다.
+
+**FastAPI** 각 경우에 수행할 작업과 동일한 개체를 내부적으로 재사용하기에, 모든 백그라운드 작업이 함께 병합되고 나중에 백그라운드에서 실행됩니다.
```
Comment (joonas-yoon) [inline docs/ko/docs/tutorial/background-tasks.md:58]: i guess you've missing `는`

**Review by joonas-yoon — COMMENTED** [review]
Thanks for your work :) I commented a little bit.

File: docs/ko/docs/tutorial/background-tasks.md (hunk 3 above)
Comment (junah201) [reply docs/ko/docs/tutorial/background-tasks.md:58]:
```suggestion
**FastAPI**는 각 경우에 수행할 작업과 동일한 개체를 내부적으로 재사용하기에, 모든 백그라운드 작업이 함께 병합되고 나중에 백그라운드에서 실행됩니다.
```

File: docs/ko/docs/tutorial/background-tasks.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+`BackgroundTasks` (`BackgroundTask`가 아닌) 만 사용하면 객체를 직접 사용할 때와 마찬가지로 _경로 동작 함수_ 의 매개변수로 사용하고, 나머지는 **FastAPI** 에서 처리하도록 할 수 있습니다. (`Request` 객체를 직접 사용하는 것과 유사하게)
+
+FastAPI에서 `BackgroundTask`를 단독으로 사용하는 것은 여전히 가능합니다. 하지만 객체를 코드에서 생성하고, 이 객체를 포함하는 Starlette `Response`를 반환해야 합니다.
+
+<a href="https://www.starlette.io/background/" class="external-link" target="_blank">`백그라운드 작업에 대한 Starlette의 공식 문서`</a>에서 더 자세한 내용을 볼 수 있습니다.
+
+## 경고
+
+만약 무거운 백그라운드 작업을 수행하야하고 동일한 프로세스에서 실행할 필요가 없는 경우 (예: 메모리, 변수 등을 공유할 필요가 없음) <a href="https://docs.celeryq.dev" class="external-link" target="_blank">`Celery`</a>와 같은 큰 도구를 사용하면 도움이 될 수 있습니다.
```
Comment (joonas-yoon) [inline docs/ko/docs/tutorial/background-tasks.md:92]: I found typo :)

```suggestion
만약 무거운 백그라운드 작업을 수행해야하고 동일한 프로세스에서 실행할 필요가 없는 경우 (예: 메모리, 변수 등을 공유할 필요가 없음) <a href="https://docs.celeryq.dev" class="external-link" target="_blank">`Celery`</a>와 같은 큰 도구를 사용하면 도움이 될 수 있습니다.
```

File: docs/ko/docs/tutorial/background-tasks.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+요청에 쿼리가 있는 경우 백그라운드 작업의 로그에 기록됩니다.
+
+그리고 _경로 동작 함수_ 에서 생성된 또 다른 백그라운드 작업은 경로 매개 변수를 활용하여 사용하여 메시지를 작성합니다.
+
+## 기술적 세부사항
+
+`BackgroundTasks` 클래스는 <a href="https://www.starlette.io/background/" class="external-link" target="_blank">`starlette.background`</a>에서 직접 가져옵니다.
+
+`BackgroundTasks` 클래스는 FastAPI에서 직접 임포트하거나 포함하기 때문에 실수로 `BackgroundTask` (끝에 s가 없음)을 임포트하더라도 starlette.background에서 `BackgroundTask`를 가져오는 것을 방지할 수 있습니다.
```
Comment (joonas-yoon) [inline docs/ko/docs/tutorial/background-tasks.md:82]: In my opinion, I think we need to keep the format style for the letter `s` in this case

```suggestion
`BackgroundTasks` 클래스는 FastAPI에서 직접 임포트하거나 포함하기 때문에 실수로 `BackgroundTask` (끝에 `s`가 없음)을 임포트하더라도 starlette.background에서 `BackgroundTask`를 가져오는 것을 방지할 수 있습니다.
```

File: docs/ko/docs/tutorial/background-tasks.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+`BackgroundTasks` (`BackgroundTask`가 아닌) 만 사용하면 객체를 직접 사용할 때와 마찬가지로 _경로 동작 함수_ 의 매개변수로 사용하고, 나머지는 **FastAPI** 에서 처리하도록 할 수 있습니다. (`Request` 객체를 직접 사용하는 것과 유사하게)
```
Comment (joonas-yoon) [inline docs/ko/docs/tutorial/background-tasks.md:84]: This is my recommendation, and..
I'm confusing how to translate "_path operation function_ parameter". This is not "_path operation function_'s parameter", neither "_path...func_ for parameter".

```suggestion
(`BackgroundTask`가 아닌) `BackgroundTasks`를 사용하면, _경로 동작 함수_ 매개변수로 사용할 수 있게 되고 나머지는 **FastAPI**가 대신 처리하도록 할 수 있습니다. 이것은 `Request` 객체를 직접 사용하는 것과 같은 방식입니다.
```

**Review by junah201 — COMMENTED** [review]
There are some minor fixes

**Comment by tiangolo** [comment]
📝 Docs preview for commit e3284b4de1c44223c3518e3d001a94ac69e27fef at: https://649a1e92acd72b160761e973--fastapi.netlify.app

File: docs/ko/docs/tutorial/background-tasks.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
+# 백그라운드 작업
+
+응답을 반환한 후 실행할 백그라운드 작업을 정의할 수 있습니다.
```
Comment (Junhyung21) [inline docs/ko/docs/tutorial/background-tasks.md:3]:
```suggestion
FastAPI에서는 응답을 반환한 후에 실행할 백그라운드 작업을 정의할 수 있습니다.
```

File: docs/ko/docs/tutorial/background-tasks.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+```Python hl_lines="1  13"
+{!../../../docs_src/background_tasks/tutorial001.py!}
+```
+
+**FastAPI** 는 `BackgroundTasks` 개체를 생성하고, 매개 변수로 전달합니다.
+
+## 작업 함수 생성
+
+백그라운드 작업으로 실행할 함수를 만듭니다.
```
Comment (Junhyung21) [inline docs/ko/docs/tutorial/background-tasks.md:26]:
```suggestion
백그라운드 작업으로 실행할 함수를 정의합니다.
```

**Review by Junhyung21 — COMMENTED** [review]
I corrected some grammatical errors and awkward parts.

약간의 문법 오류 및 어색한 부분을 수정하였습니다.

File: docs/ko/docs/tutorial/background-tasks.md [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+
+`BackgroundTasks` 클래스는 <a href="https://www.starlette.io/background/" class="external-link" target="_blank">`starlette.background`</a>에서 직접 가져옵니다.
+
+`BackgroundTasks` 클래스는 FastAPI에서 직접 임포트하거나 포함하기 때문에 실수로 `BackgroundTask` (끝에 `s`가 없음)을 임포트하더라도 starlette.background에서 `BackgroundTask`를 가져오는 것을 방지할 수 있습니다.
+
+(`BackgroundTask`가 아닌) `BackgroundTasks`를 사용하면, _경로 동작 함수_ 매개변수로 사용할 수 있게 되고 나머지는 **FastAPI**가 대신 처리하도록 할 수 있습니다. 이것은 `Request` 객체를 직접 사용하는 것과 같은 방식입니다.
+
+FastAPI에서 `BackgroundTask`를 단독으로 사용하는 것은 여전히 가능합니다. 하지만 객체를 코드에서 생성하고, 이 객체를 포함하는 Starlette `Response`를 반환해야 합니다.
+
+<a href="https://www.starlette.io/background/" class="external-link" target="_blank">`백그라운드 작업에 대한 Starlette의 공식 문서`</a>에서 더 자세한 내용을 볼 수 있습니다.
```
Comment (Junhyung21) [inline docs/ko/docs/tutorial/background-tasks.md:88]:
```suggestion
<a href="https://www.starlette.io/background/" class="external-link" target="_blank">`Starlette의 공식 문서`</a>에서 백그라운드 작업에 대한 자세한 내용을 확인할 수 있습니다.
```

File: docs/ko/docs/tutorial/background-tasks.md [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+
+**FastAPI** 는 `BackgroundTasks` 개체를 생성하고, 매개 변수로 전달합니다.
+
+## 작업 함수 생성
+
+백그라운드 작업으로 실행할 함수를 만듭니다.
+
+이것은 단순히 매개변수를 받을 수 있는 표준 함수일 뿐입니다.
+
+FastAPI는 이것이 `async def` 함수이든, 일반 `def` 함수이든 내부적으로 이를 올바르게 처리합니다.
```
Comment (Junhyung21) [inline docs/ko/docs/tutorial/background-tasks.md:30]:
```suggestion
**FastAPI**는 이것이 `async def` 함수이든, 일반 `def` 함수이든 내부적으로 이를 올바르게 처리합니다.
```

**Review by Junhyung21 — APPROVED** [review]
Approve !

**Comment by alejsdev** [comment]
Thank you! @junah201 🚀 

And thanks for the reviews @joonas-yoon and @Junhyung21 🧐🔍

**Pull request merged**